pkg/operator/controller/ingress/deployment.go

```diff
@@ -592,6 +592,21 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		nodeSelector["node-role.kubernetes.io/master"] = ""
 	default:
 		nodeSelector["node-role.kubernetes.io/worker"] = ""
+		// Disabling running on remote workers
```

Comment:
```suggestion
		// Disabling running on remote workers.
```
Is there any intention to allow scheduling routers on remote workers in some future release?